docs/core/deploying/native-aot/index.md

```diff
@@ -196,4 +141,18 @@ The following table shows supported compilation targets.
 | MacCatalyst     | x64, Arm64 | Experimental support                   |
 | Android | x64, Arm64 | Experimental, no built-in Java interop |
 
+### [.NET 9+](#tab/net9plus)
+
+| Platform | Supported architecture |  Notes               |
+|----------|------------------------|----------------------|
+| Windows  | x64, Arm64, x86        |                      |
+| Linux    | x64, Arm64, Arm        |                      |
+| macOS   | x64, Arm64 |                                        |
+| iOS     | Arm64 | Experimental support                   |
+| iOSSimulator     | x64, Arm64 | Experimental support                   |
+| tvOS     | Arm64 | Experimental support                   |
+| tvOSSimulator     | x64, Arm64 | Experimental support                   |
+| MacCatalyst     | x64, Arm64 | Experimental support                   |
```

Comment:
```suggestion
| iOS     | Arm64 |                   |
| iOSSimulator     | x64, Arm64 |                   |
| tvOS     | Arm64 |                    |
| tvOSSimulator     | x64, Arm64 |                    |
| MacCatalyst     | x64, Arm64 |                   |
```

I think this comment should be removed, as all the work on supporting NativeAOT with MAUI apps targeting iOS platforms has been completed and we will officially support it in .NET 9 release.